caom2utils/caom2utils/fits2caom2.py

```diff
@@ -2788,10 +2905,20 @@ def _get_coord_error(self, index):
         aug_coord_error = None
         aug_csyer = self._sanitize(self.wcs.csyer[index])
         aug_crder = self._sanitize(self.wcs.crder[index])
-        if aug_csyer and aug_crder:
+        if aug_csyer is not None and aug_crder is not None:
             aug_coord_error = CoordError(aug_csyer, aug_crder)
         return aug_coord_error
 
+    def _get_coord_range_1d(self, index):
+        # TODO - is this the right thing? It doesn't seem so, based on
+        # the keywords involved .... ugh
+        aug_coord_range = None
+        aug_start = self._get_ref_coord(index)
+        aug_end = self._get_ref_coord(index)
```

Comment:
It is. I wrote this function before Pat said that FITS WCS only covers values for CoordFunction1D and CoordFunction2D, not CoordRange* or CoordBounds*. So I've removed this function from the code too, since fits2caom2 is scoped to WCS-derivable content.